### PR TITLE
MAN-93 create collection entity model

### PR DESCRIPTION
**Pull request by nomadicoder**

### CODE REVIEW

**Comment by nomadicoder** [comment]
@cdoyle-temple Would you have a look at the administrate for `collections`, then pass it back to me and I'll finish it up?

**Comment by nomadicoder** [comment]
Work in progress. Do not merge.

File: app/views/fields/building_field/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="field-unit__label">
```
Comment (bibliotechy) [inline app/views/fields/building_field/_form.html.erb]: Do we still need this?

File: app/views/fields/building_field/_index.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= field.data.name %>
```
Comment (bibliotechy) [inline app/views/fields/building_field/_index.html.erb]: Do we still need this?

File: app/views/fields/building_field/_show.html.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<% unless field.data.nil? %>
```
Comment (bibliotechy) [inline app/views/fields/building_field/_show.html.erb]: Do we still need this?

File: app/views/fields/building_field/_form.html.erb (hunk 1 above)
Comment (nomadicoder) [reply app/views/fields/building_field/_form.html.erb]: Done

File: app/views/fields/building_field/_show.html.erb (hunk 3 above)
Comment (nomadicoder) [reply app/views/fields/building_field/_show.html.erb]: Deleted

File: app/views/fields/building_field/_index.html.erb (hunk 2 above)
Comment (nomadicoder) [reply app/views/fields/building_field/_index.html.erb]: Deleted

**Pull request merged**